core/src/main/java/apoc/export/util/ExportConfig.java

```diff
@@ -99,6 +100,7 @@ public ExportConfig(Map<String,Object> config) {
         this.optimizations = (Map<String, Object>) config.getOrDefault("useOptimizations", Collections.emptyMap());
         this.optimizationType = OptimizationType.valueOf(optimizations.getOrDefault("type", OptimizationType.UNWIND_BATCH.toString()).toString().toUpperCase());
         this.batchSize = ((Number)config.getOrDefault("batchSize", DEFAULT_BATCH_SIZE)).intValue();
+        this.sample = Util.toLong(config.getOrDefault("sample", DEFAULT_BATCH_SIZE));
```

Comment:
questo non serve più giusto?

Reply:
pensandoci potremmo sfruttarlo per metterci dentro tutta la configurazione del sampling
```suggestion
        this.samplingConfig = Util.toLong(config.getOrDefault("samplingConfig", Collections.emptyMap()));
```